code/game/objects/items/weapons/RCD.dm

```diff
@@ -12,8 +12,8 @@ RCD
 	opacity = 0
 	density = 0
 	anchored = 0
-	flags = CONDUCT
-	force = 10
+	flags = CONDUCT | NOBLUDGEON
+	force = 0
```

Comment:
Super controversial: can't hit people with the RCD anymore. It's the only viable solution I found to make using the RCD on an airlock not open it.